projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py

```diff
@@ -248,7 +248,7 @@ def _login(self) -> None:
             token=self.conn.extra_dejson.get("token", None),
             authorization_url=self.conn.extra_dejson.get("auth_server", None),
             auth_type=self.conn.extra_dejson.get("auth_type", None),
-            cache_locally=True,
+            cache_locally=False,
```

Comment:
Why changing this?

Reply:
It is part of another change I am working on. Will document it in the next PR.